components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -328,6 +328,10 @@ func generateStatefulSet(instance *v1beta1.Notebook) *appsv1.StatefulSet {
 			FSGroup: &fsGroup,
 		}
 	}
+	schedulerName := os.Getenv("SCHEDULER_NAME")
+	if len(schedulerName) > 0 {
+		podSpec.SchedulerName = schedulerName
+	}
```

Comment:
Don' t think this is a good way to configure the scheduler in notebook controller. We should control this through the notebook CR itself